core/src/main/resources/db/cgds.sql

```diff
@@ -724,8 +727,34 @@ CREATE TABLE `clinical_event_data` (
   FOREIGN KEY (`CLINICAL_EVENT_ID`) REFERENCES `clinical_event` (`CLINICAL_EVENT_ID`) ON DELETE CASCADE
 );
 
+CREATE TABLE `gene_panel` (
+    `INTERNAL_ID` int(11) NOT NULL auto_increment,
+    `STABLE_ID` varchar(255) NOT NULL,
+    `DESCRIPTION` mediumtext,
+    PRIMARY KEY (`INTERNAL_ID`),
+    UNIQUE (`STABLE_ID`)
+);
+
+CREATE TABLE `gene_panel_list` (
+    `INTERNAL_ID` int(11) NOT NULL,
+    `GENE_ID` int(255) NOT NULL,
+    PRIMARY KEY (`INTERNAL_ID`, `GENE_ID`),
+    FOREIGN KEY (`INTERNAL_ID`) REFERENCES `gene_panel` (`INTERNAL_ID`) ON DELETE CASCADE,
+    FOREIGN KEY (`GENE_ID`) REFERENCES `gene` (`ENTREZ_GENE_ID`) ON DELETE CASCADE
+);
+
+CREATE TABLE `gene_panel_profile_map` (
+    `SAMPLE_ID` int(11) NOT NULL,
+    `PROFILE_ID` int(11) NOT NULL,
+    `PANEL_ID` int(11) NOT NULL,
+    PRIMARY KEY (`SAMPLE_ID`, `PROFILE_ID`),
```

Comment:
Furthermore: according to File-Formats.md, metadata files for genetic profiles specify one gene panel for the entire profile. Why can `PANEL_ID` not just be added to the `genetic_profile` table then? Will different samples in the same genetic profile (~experiment/data file) be targeted using different gene panels?